src/twisted/conch/ssh/transport.py

```diff
@@ -464,6 +465,7 @@ class SSHTransportBase(protocol.Protocol):
     supportedMACs = [
         b'hmac-sha2-512',
         b'hmac-sha2-256',
+        b'hmac-sha2-384',
```

Comment:
I think that this should be just after hmac-sha2-512 as hmac-sha2-384 should be preferred over hmac-sha2-256 ...

see the comment about having this list based on preference